composer.json

```diff
@@ -24,41 +24,40 @@
         "doctrine/common": "^2.7",
-        "doctrine/inflector": "^1.1",
-        "knplabs/knp-menu": "^3.1",
-        "knplabs/knp-menu-bundle": "^3.0",
-        "sonata-project/block-bundle": "^4.0",
-        "sonata-project/doctrine-extensions": "^1.4",
-        "sonata-project/exporter": "^2.1",
-        "sonata-project/form-extensions": "^1.1.2",
-        "sonata-project/twig-extensions": "^1.1.1",
-        "symfony/asset": "^4.3",
-        "symfony/config": "^4.3",
-        "symfony/console": "^4.3",
-        "symfony/dependency-injection": "^4.3",
-        "symfony/doctrine-bridge": "^4.3",
-        "symfony/event-dispatcher": "^4.3",
+        "doctrine/inflector": "^1.4 || ^2.0",
+        "knplabs/knp-menu-bundle": "^3.0.0",
+        "sonata-project/block-bundle": "^4.2",
+        "sonata-project/doctrine-extensions": "^1.6",
+        "sonata-project/exporter": "^1.11.0 || ^2.0",
+        "sonata-project/form-extensions": "^1.3",
+        "sonata-project/twig-extensions": "^1.2",
+        "symfony/asset": "^4.3 || ^5.0",
+        "symfony/config": "^4.3 || ^5.0",
+        "symfony/console": "^4.3 || ^5.0",
+        "symfony/dependency-injection": "^4.3 || ^5.0",
+        "symfony/doctrine-bridge": "^4.3 || ^5.0",
+        "symfony/event-dispatcher": "^4.3 || ^5.0",
         "symfony/event-dispatcher-contracts": "^1.1 || ^2.0",
-        "symfony/expression-language": "^4.3",
-        "symfony/form": "^4.3",
-        "symfony/framework-bundle": "^4.3",
-        "symfony/http-foundation": "^4.3",
-        "symfony/http-kernel": "^4.3",
-        "symfony/options-resolver": "^4.3",
-        "symfony/property-access": "^4.3",
-        "symfony/routing": "^4.3",
+        "symfony/expression-language": "^4.3 || ^5.0",
+        "symfony/form": "^4.0 || ^5.0",
+        "symfony/framework-bundle": "^4.3 || ^5.0",
+        "symfony/http-foundation": "^4.3 || ^5.0",
+        "symfony/http-kernel": "^5.0",
+        "symfony/options-resolver": "^4.3 || ^5.0",
+        "symfony/property-access": "^4.3 || ^5.0",
+        "symfony/routing": "^4.3 || ^5.0",
         "symfony/security-acl": "^3.0",
-        "symfony/security-bundle": "^4.3",
-        "symfony/security-core": "^4.3",
-        "symfony/security-csrf": "^4.3",
-        "symfony/translation": "^4.3",
-        "symfony/twig-bridge": "^4.3",
-        "symfony/twig-bundle": "^4.3",
-        "symfony/validator": "^4.3",
-        "twig/extensions": "^1.5",
+        "symfony/security-bundle": "^4.3 || ^5.0",
+        "symfony/security-core": "^4.3 || ^5.0",
+        "symfony/security-csrf": "^4.3 || ^5.0",
+        "symfony/templating": "^4.3 || ^5.0",
+        "symfony/translation": "^4.3 || ^5.0",
+        "symfony/twig-bridge": "^4.3 || ^5.0",
+        "symfony/twig-bundle": "^4.3 || ^5.0",
+        "symfony/validator": "^4.3 || ^5.0",
         "twig/extra-bundle": "^3.0",
         "twig/string-extra": "^3.0",
-        "twig/twig": "^2.12.1"
+        "twig/twig": "^2.12.1 || ^3.0.0"
```

Comment:
For twig 3, there is https://github.com/sonata-project/SonataAdminBundle/pull/5973

Maybe you should focus on symfony 5.

Reply:
Do you want i rollback changes related to Twig 3 ?

Reply:
I think keeping `twig/twig": "^2.12.1` and not trying to add support for twig 3 here will make the PR easier.

Adding support for twig 3 and adding support for Symfony 5 should be able to be done in two separate PR.

Reply:
ok